### DISCUSSION TITLE
What Dataset can i Validate this Modell on?

### DISCUSSION THREAD

**Discussion by NiklasHoltmeyer** [snakers4/silero-models#67] [Q&A] [answered]
Hi Guys, 

i couldnt find what your German Model is trained on, i could just find that you dont want to answer that question. Could you maybe tell me, what Dataset i can use to validate this Modell -> ergo what Dataset you dont have trained this Modell on?

Can i just use CommonVoice-Test-Split or Libri-Speech-Test?

Ty in advanced

**Comment by snakers4** [comment, marked as answer]
>  Could you maybe tell me, what Dataset i can use to validate this Modell -> ergo what Dataset you dont have trained this Modell on?

You can find the detailed quality benchmarks for the German model here - https://github.com/snakers4/silero-models/wiki/Quality-Benchmarks#de-v1

**Comment by NiklasHoltmeyer** [comment]
I wanted to compare different data augmentation techniques and for validation I need data sets that the network does not know yet. I have them now, if I can assume that the listed data sets are still unknown to the network. Thanks

**Reply by snakers4** [reply]
Technically for networks trained on large data (i.e. 5-10k hours and higher) train / test split does not really matter

Only the presence of the necessary domain in the train data matters 

Hence you can just validate on your domain

Benchmarks on public datasets are done for reference
